### DISCUSSION TITLE
Does it really make difference 5900x 2x 2tb 980 pro 40 gb Ram

### DISCUSSION THREAD

**Discussion by nem0n** [madMAx43v3r/chia-plotter#263] [General]
Hi Guys,

So I was looking into this project and trying to understand if it is really benefical to my case. Currently on windows I am plotting.
Here is how its going approx I am getting 27-30 plot per day.

![image](https://user-images.githubusercontent.com/25449049/121715301-5e1ac180-cae7-11eb-9428-29887307f5d5.png)

So my question is with my setup switching to madmax would make difference ? or should I just stick with it.

Thank you

**Comment by Dartellum** [comment]
Try it out. 😀

**Reply by nem0n** [reply]
I want to :D but if I stop the current ones just for testing will be taste of 15-20 plot already so wondered if someone did it

**Comment by thiagottjv** [comment]
@nem0n nem0n, with a similar setup (5900x, 32gb RAM, 2x 970 2Tb, 2 external USB3 HDs and clearlinux) I was getting around 40 to 42 plots.
I just finished testing this and each plot is taking around 1750s, witch mean that I`ll do around 49 plots a day using only one HD.
The best thing about this plotter is the same thing that you don`t want to stop yours: you`ll lose the current plots. With this you just lose one and you dont need 3 end HDs to lower the copy times. No need to figure out staggering times and so on.

So answering your question. Yes, I think it would make a difference.

P.S.: I don`t know witch system you`re using, but if youre using windows you should take something around 10 to 15% off what I just said. If you`re using a gui distro you should take around 5% or so.

**Reply by nem0n** [reply]
@thiagottjv first of all thank you for the feedback. The thing is currently I have one 256 gb 980 pro and 2 tb due to my system is installed I was not able to connect the other 2 tb that I have so waiting for converter. But after posting this I decided to give it a try to mad max and it took 55 minute to get the plot done which approxtmly will be 25-26 plot maybe as a result of end of the day. But with swar I am able to get around 27-30 plot and hoping that with another 2 tb it will increase. But so far I didnt see the advantage of the mad max.

And yes I am using windows because of this is my main machine and have many stuff installed on it and gpu mining at the same time.

**Reply by thiagottjv** [reply]
You could do what I did. I partitioned my nvme with 100Gb for the OS and the rest for plotting. With this I have 1.7Tb with one 970 pro and another with the OS instalation and more 1.6 for plotting.
The next test I`ll do is plot for  windows. Since I'll have 64Gb and I'm using only around 20Gb with this plotter I`ll try to use 23 threads (instead of 24) alongside windows and compare the times.

**Comment by Dschijn** [comment]
6 in parallel on a 5900x isn't that great, look at your CPU and RAM usage. You could do at least 10, maybe 12 in parallel staggered with SWAR.

**Reply by nem0n** [reply]
Generally I am doing parallel for 7-8 plot with current delay. But because I have kind of 1 x 2 tb nvme and 1x 256 gb I think that becoming my limitation and thats why if I try to go over that I am ending up with 20 plot or maybe even less. With my current approach I am getting around 27-30 plot. But not sure if this is good enough for my setup currently

**Reply by Dschijn** [reply]
True. I am pretty sure, with two propper 2TB SSDs you could do 12 staggered. 40GB of RAM might be an issue in Windows though.
I got a 14core Intel CPU and 2x2TB plotting in Ubuntu with SWAR, 14 in parallel and up to 16 when starting 1-2 early. Manage 44-46 plots per day that way.
Not sure if madmax could help me, since I don't habe enough memory in that system.

**Reply by nem0n** [reply]
I tried madmax but apperaently it taking 55 min for one plot which is compare swar not better

**Reply by thiagottjv** [reply]
> I tried madmax but apperaently it taking 55 min for one plot which is compare swar not better

Are you using all of the 24 threads?

**Reply by nem0n** [reply]
Yes I tried use all of my threads but my second temp was not ram it was another ssd which is 256 and which result of 55 min

**Comment by casa10** [comment]
Can you share your configuration of swar ? (if you want of course)

**Reply by nem0n** [reply]
![image](https://user-images.githubusercontent.com/25449049/121775745-24eb5b80-cb92-11eb-81f7-413745550946.png)

Here you go

**Reply by Dschijn** [reply]
consider using "max_concurrent_with_start_early" with one more plot than your 9 max_concurrent. Maybe with a short delay within phase 4 of 5-10min.
By that a new plot will already start shortly before phase 4 is finished and only phase 5 is copying the plot (which is not CPU intensiv).

**Reply by casa10** [reply]
Thank you.
i am surprised : you make 27-30 plots by day with 3 threads using only one NVME ? and one NVME?
@thiagottjv : you do 40-42 plots by days with your configuration withour using this new way to plot? can you share your configuration of swar please (if you accept) ?

@Dschijn
 : same question

**Reply by nem0n** [reply]
@Dschijn I didnt play with that feature much but for sure will give it a try

**Reply by nem0n** [reply]
@casa10 yeah I do around 27-30 after it settled what I mean by settled  is that like when you stop it until its fully back up and plotting in pararel. When it reaching to that stage then yeah I am getting around that but will try even more increase it

**Reply by Dschijn** [reply]
My config for an Intel i9 9940x with 14 cores, pltting on two Intel SSD DC P4510:
global:
  max_concurrent: 16
  max_for_phase_1: 8
  minimum_minutes_between_jobs: 10

jobs:
  - name: first
    max_plots: 999
    farmer_public_key: xxx
    pool_public_key: xxx
    temporary_directory: /media/ssd1/chiatemp
    temporary2_directory:
    destination_directory: /media/tower/chiatemp1
    size: 32
    bitfield: true
    threads: 4
    buckets: 128
    memory_buffer: 5000
    max_concurrent: 7
    max_concurrent_with_start_early: 8
    initial_delay_minutes: 0
    stagger_minutes: 30
    max_for_phase_1: 3
    concurrency_start_early_phase: 4
    concurrency_start_early_phase_delay: 5
    temporary2_destination_sync: false
    exclude_final_directory: false
    skip_full_destinations: true
    unix_process_priority: 10
    windows_process_priority: 32
    enable_cpu_affinity: true
    cpu_affinity: [ 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13 ]

  - name: second
    max_plots: 999
    farmer_public_key: xxx
    pool_public_key: xxx
    temporary_directory: /media/ssd2/chiatemp
    temporary2_directory:
    destination_directory: /media/tower/chiatemp2
    size: 32
    bitfield: true
    threads: 4
    buckets: 128
    memory_buffer: 5000
    max_concurrent: 7
    max_concurrent_with_start_early: 8
    initial_delay_minutes: 20
    stagger_minutes: 30
    max_for_phase_1: 3
    concurrency_start_early_phase: 4
    concurrency_start_early_phase_delay: 5
    temporary2_destination_sync: false
    exclude_final_directory: false
    skip_full_destinations: true
    unix_process_priority: 10
    windows_process_priority: 32
    enable_cpu_affinity: true
    cpu_affinity: [ 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27 ]
